adaptdl_cli/bin/adaptdl

```diff
@@ -320,6 +348,9 @@ if __name__ == "__main__":
     parser_submit.add_argument(
         "--checkpoint-storage-class", type=str,
         help="storage class of new temporary pvc to create", default=None)
+    parser_submit.add_argument(
+        "--proxy-port", type=int, default=59283,  # Some uncommon port.
```

Comment:
Not sure why this an argument and not just any available port (using say portpicker)?

Reply:
If we pick a different port for each `adaptdl submit` then it will leave many temporary image tags hanging around in `docker images`. Also in the MacOS case it will require modifying daemon.json and restarting docker each time `adaptdl submit` is run.

Reply:
In case the default port is not available on localhost, the user has to choose an available port. Could they use the same `proxy-port` for multiple successive `adaptdl submit` runs without getting port bind errors (from the proxy)?

Reply:
Yes